packages/components/src/date-time/date/styles.ts

```diff
@@ -17,12 +17,24 @@ export const Wrapper = styled.div`
 `;
 
 export const Navigator = styled( HStack )`
+	display: grid;
+	grid-template-columns: 0.5fr repeat( 5, 1fr ) 0.5fr;
+	justify-items: center;
```

Comment:
I wonder if we should specify `column-gap` to guarantee equal spacing with `Calendar` component below, if the intention is to simulate the same grid as the calendar days? It looks like this inherits a default column gap from `HStack`, but that feels like a pretty loose guarantee.